common/src/main/java/com/microsoft/identity/common/internal/broker/ipc/BoundServiceStrategy.java

```diff
@@ -16,7 +16,7 @@
 import java.util.concurrent.TimeoutException;
 
 /**
- * A strategy for communicating with the active broker (:auth process) via Bound Service.
+ * A strategy for communicating with the targeted broker via Bound Service.
```

Comment:
How is the broker going to be specified?

Reply:
BrokerOperationBundle.targetBrokerAppPackageName.